src/essential_validator.py

```diff
@@ -262,8 +269,9 @@ def validate_data(self, file_info):
             return False
         
         # Each row in a metadata file must have same number of columns as the header row
-        if '' in self.df.columns:
-            msg = f'Invalid metadata, headers are match row columns, {self.batch[ID]}!'
+        empty_cols = [col for col in self.df.columns.tolist() if "Unnamed:" in col or not col]
```

Comment:
Where is "Unnamed" string coming from? Can we avoid this? If not, we should create a constant variable. And, we should leave a comment.

Reply:
during dataframe parsing the tsv file.